web/html/src/manager/systems/activation-key/activation-key-channels.renderer.js

```diff
@@ -10,6 +11,6 @@ window.pageRenderers = window.pageRenderers || {};
 const customValues = window.pageRenderers.customValues || {DOMid: 'activation-key-channels'};
 
 export const renderer = () => SpaRenderer.renderNavigationReact(
```

Comment:
just adapt this function:

`
export const renderer = (id, {activationKeyId}) => SpaRenderer.renderNavigationReact(
`

Use the function parameters values instead of the global ones